client/components/select-dropdown/test/index.js

```diff
@@ -51,6 +51,31 @@ describe( 'index', () => {
 			toggleDropdownStub.restore();
 		} );
 
+		test( 'should not respond when clicked when disabled', () => {
+			const toggleDropdownSpy = sinon.spy( SelectDropdown.prototype, 'toggleDropdown' );
+
+			const dropdown = shallowRenderDropdown( {
+				disabled: true,
+			} );
+
+			expect( dropdown.find( '.select-dropdown.is-disabled' ) ).to.have.lengthOf( 1 );
+
+			dropdown.find( '.select-dropdown__container' ).simulate( 'click' );
+
+			sinon.assert.called( toggleDropdownSpy );
+
+			expect( dropdown.find( '.select-dropdown.is-open' ) ).to.be.empty;
+
+			// Repeat to be sure
```

Comment:
Why wouldn't we be sure if the dropdown is disabled here?

Reply:
@ehg Because if the dropdown was already open when the `disabled` prop changed then it might remain open. I've updated the code to cater for this by making the application of the `is-open` class dependent on `disabled` being false.